make/migrations/postgresql/0031_2.0.1_schema.up.sql

```diff
@@ -0,0 +1,4 @@
+/*remove the constraint for name in table 'notification_policy'*/
+ALTER TABLE notification_policy DROP CONSTRAINT notification_policy_name_key;
```

Comment:
You should add the sql into the sql file for 2.1: https://github.com/goharbor/harbor/blob/master/make/migrations/postgresql/0040_2.1.0_schema.up.sql

Reply:
fixed